sdk/apimanagement/api-management-custom-widgets-scaffolder/package.json

```diff
@@ -41,7 +41,7 @@
     "build:browser": "tsc -p . && dev-tool run bundle --browser-test=false",
     "build:node": "tsc -p . && dev-tool run bundle",
     "build:test": "echo skip",
-    "bundle:bin": "rollup -c rollup.config.bin.js",
+    "bundle:bin": "rollup -c rollup.config.bin.mjs",
```

Comment:
curious why .mjs is needed now. because of the import assert below?

Reply:
yeah rollup v3 is strict about the difference between cjs and mjs.